File: src/main/java/model/Move.java
package model;

public class Move {

    private Tile tile;
    private Position position;

    public Move(Tile tile, Position position) {
        this.tile = tile;
        this.position = position;
    }

    public Tile getTile() {
        return tile;
    }

    public Position getPosition() {
        return position;
    }
}

File: docs/scripts/model/Move.js
export default class Move {

    constructor(tile, position) {
        this.tile = tile;
        this.position = position;
    }

    getTile() {
        return this.tile;
    }

    setTile(tile) {
        this.tile = tile;
    }

    getPosition() {
        return this.position;
    }

    setPosition(position) {
        this.position = position;
    }
}
